docs/source/releases/v1_11_5a0.rst

```diff
@@ -38,3 +39,68 @@ Update "update_this_release_note" to docs/source/releases/v1_11_5a0.rst
 ::
 
   modified: update_this_release_note
+
+
+Code Checks
+===========
+
+Fix Errors from Flake8 Linting
+------------------------------
+
+*From issue GEOIPS#298: 2023-08-26, Fix remaining flake8 errors / turn on full testing*
+
+GeoIPS has been ignoring flake8 errors when running the check_code.sh script. This needs
```

Comment:
@evrose54 can you please run "full_test.sh" and paste the output into the PR contents?  I'd like to see that all the actual integration tests still pass after the updates.

Reply:
working on this now

Reply:
added!